docs/sources/enterprise/usage-insights.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Usage-insights"
+description = "Usage-insights"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/"]
+type = "docs"
+[menu.docs]
+name = "Usage-insights"
+parent = "enterprise"
+weight = 700
++++
+
+# Usage Insights
+
+Usage Insights allows you to have a better understanding on how your Grafana instance is used. The collected data are: dashboard views (aggregated and per user), data source errors, and number of data source queries.
+
+> Only available in Grafana Enterprise v7.0+.
+
+## Presence Indicators
+
+Presence indicators are visible to all signed-in users on all dashboards. It shows avatar of users who interacted with the dashboard recently. You can see the user's name by hovering the user's avatar. Users' avatars come from [Gravatar](https://gravatar.com) based on the user's email.
+
+When more users are active on a dashboard than can fit in the presence indicators, click on the `+X` icon that opens the [Dashboard Insights]({{< relref "#dashboard-insights" >}}) drawer and see more details about recent user activity. 
+
+{{< docs-imagebox img="/img/docs/enterprise/presence_indicators.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+You can choose your own definition of "recent" by setting it in the [configuration]({{< relref "../installation/configuration.md">}}) file.
+
+```ini
+[analytics.views]
+# Set age for recent active users
+recent_users_age = 24h
+```
+
+## Dashboard Insights
+
+You can see dashboard usage information by clicking on the `Dashboard insights` button in the top bar.
```

Comment:
What button is this? Is there an image or can you provide a description?

Reply:
I added an image with the button highlighted